### ISSUE TITLE
New Config - Should `force_tx_in` work with `disable` option for `tx_out` ?

### ISSUE THREAD

**Issue by ArturWieczorek** [IntersectMBO/cardano-db-sync#1789]
Should this config:

```json
  "insert_options": {
     "tx_out": {
        "value": "disabled",
        "force_tx_in": true
     },
  },
```
work with `disable` option and force data into `tx_in` table ? 
I think previously it was reserved for `consumed_tx_out` flag only but I might be wrong.

The current situation is that the config above would not for data to appear inside `tx_in` table.

**Comment by kderme** [comment]
I think it's just ignored with enabled or disabled. Probably something we want to document.

**Comment by sgillespie** [comment]
> I think it's just ignored with enabled or disabled. Probably something we want to document.

That's right, `disabled` completely supercedes it. We should add a caveat here: https://github.com/IntersectMBO/cardano-db-sync/blob/master/doc/configuration.md#force-tx-in
